setup/generate_upstream_rust_keylime_code_coverage/test.sh

```diff
@@ -15,7 +15,7 @@ rlJournalStart
         #delete coverage script, in dir are mandatory only coverage files
         rlRun "pushd ${__INTERNAL_limeCoverageDir}"
         # export in format for user, gather code cover for all rust binaries
-        rlRun "source /root/.cargo/env"
+        rlRun "export \"PATH=${PATH}:/root/.cargo/env\""
```

Comment:
Perhaps you should check whether `/root/.cargo` exists before doing this?

Reply:
Sure, will add the check